MANUAL_REGRESSION.md

```diff
@@ -97,7 +97,10 @@ Given user is on upload document page on desktop browser
     - user should be able to upload a document from a mobile device
 
 ### 5. Cross-device with SMS in Spanish
-(on one of the desktop browsers and both Android Chrome and iOS Safari mobile browsers)
+(on private mode of one of the desktop browsers and both Android Chrome and iOS Safari mobile browsers)
+
+that doesn't have media recorder API support (Chrome on iOS)
```

Comment:
I don't understand this sentence. It doesn't have a subject and it starts with "that" without a capital. I find it confusing so I think our crowd testers would find it difficult to understand it too.

Reply:
Oops. Looks like some of the text got lost there. Will fix.